### PR TITLE
[sentry-native] update to 0.7.6

### PR DESCRIPTION
**Pull request by AenBleidd**

### CODE REVIEW

File: ports/sentry-native/fix-config-cmake.patch [hunk 1]
```diff
@@ -2,42 +2,20 @@ diff --git a/sentry-config.cmake.in b/sentry-config.cmake.in
...
- 		pkg_check_modules(BREAKPAD REQUIRED IMPORTED_TARGET breakpad-client)
- 	endif()
- endif()
-@@ -23,11 +25,9 @@ endif()
+@@ -23,7 +23,5 @@ endif()
  include("${CMAKE_CURRENT_LIST_DIR}/sentry-targets.cmake")
 
  if(SENTRY_TRANSPORT STREQUAL "curl" AND (NOT @BUILD_SHARED_LIBS@ OR NOT SENTRY_BUILD_SHARED_LIBS))
--	find_package(CURL REQUIRED)
+ 	find_dependency(CURL REQUIRED)
```
Comment (dg0yt) [inline ports/sentry-native/fix-config-cmake.patch:18]:
```suggestion
 	find_dependency(CURL)
```

**Review by dg0yt — CHANGES_REQUESTED** [review]
`find_dependency` needs `CMakeFindDependencyMacro`, and it must be used *before* including the exported targets. Can you please check these prerequisites?

(We might just leave the original `find_package(foo REQUIRED)` to reduce patching - if a dependency isn't found, the port is broken.)

File: ports/sentry-native/fix-config-cmake.patch [hunk 2]
```diff
@@ -2,42 +2,20 @@ diff --git a/sentry-config.cmake.in b/sentry-config.cmake.in
 index 70ce7d3..796d428 100644
 --- a/sentry-config.cmake.in
 +++ b/sentry-config.cmake.in
-@@ -1,5 +1,7 @@
- @PACKAGE_INIT@
+@@ -9,7 +9,7 @@ set(SENTRY_LINK_PTHREAD @SENTRY_LINK_PTHREAD@)
 
-+include(CMakeFindDependencyMacro)
```
[the remark this answers is not in the document]
Comment (AenBleidd) [reply ports/sentry-native/fix-config-cmake.patch:8]: Yes

File: ports/sentry-native/fix-config-cmake.patch (hunk 1 above)
Comment (AenBleidd) [reply ports/sentry-native/fix-config-cmake.patch:18]: The same as above

File: ports/sentry-native/fix-config-cmake.patch (hunk 2 above)
[the remark this answers is not in the document]
Comment (AenBleidd) [reply ports/sentry-native/fix-config-cmake.patch:8]: Reference: https://github.com/getsentry/sentry-native/pull/1008

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**